little-books/src/components/card/blog-card/blog-card.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { blogActions } from '../../../store/blog-store';
+import styles from './blog-card.module.scss';
+
+
+const BlogCard = ({title,type,details,photo,isSelected})=>{
+
+    const blogData = {
+        title:title,
+        details:details,
+        photo:photo,
+        type:type
+    }
+    const isEditModeOn = useSelector((state:any)=>state.editBlogState);
```

Comment:
What is use of using TS here as you given 'any' to all the state functions?

Reply:
all the types has specified with needed types and any has been replaced